cms/djangoapps/contentstore/api/views.py

```diff
@@ -101,6 +101,8 @@ class CourseImportView(CourseImportExportViewMixin, GenericAPIView):
         }
 
     """
+    exclude_from_schema = True
```

Comment:
Can you add a TODO comment with an explanation for each of the excluded APIs in this PR?  Or, is there a single common reason for excluding them?  The TODO comment can include a link to the JIRA ticket for fixing it.

Reply:
Sounds good.

What project would I create those JIRA tickets in? Is this something the arch-team would take on?

These need to be excluded because they do not specify a Serializer. The swagger tooling requires a DRF Serializer class to be specified so that it can perform introspection. Also, most of these endpoints don't look particularly REST-ful meaning they don't really deal in resources.

Reply:
@nasthagiri Any thoughts on the above questions?

Reply:
@douglashall For now, I think a single JIRA ticket (sure in the Arch space) that explains why this was done and how to find all exceptions - is fine for now.  The situation may change soon anyway as we figure out our path forward.